INZFS.MVC/INZFS.json

```diff
@@ -447,6 +447,175 @@
         "continueButtonText": "Save and continue to next question",
         "RazorView": "FinanceProposalOverview",
         "pages": [
+          {
+            "name": "subsidy-requirements",
+            "fieldType": "gdsStaticPage",
+            "question": "6.1 Funding levels and subsidy requirements",
+            "nextPageName": "subsidy-sole-applicant"
```

Comment:
We probably do not need this. Next question under pages collection should have url attached to it. We already have the code in place for this

Reply:
I think we do need this. Because the static pages aren't forms, they aren't submitted and no ContentType is saved, so the controller isn't involved in selecting the next page. Is there a better way of doing this?